### PR TITLE
Portfolio - Desktop version

### PR DESCRIPTION
**Pull request by Elerqsousy**
## Add Desktop media query and amend HTML

In this milestone:
- Added required images and backgrounds
- Added HTML components to match differences in content between Desktop version and Mobile Version
- Used media query to style the different versions
- Added desktop styles

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,36 +25,52 @@
...
+          <button class="buttons-primary-dark" type="button">
+            Start collaboration
+          </button>
+        </article>
+        <img
+          class="page-brk-dp"
+          src="images/image-geometry_1.svg"
+          alt="page break"
+        />
       </section>
```
Comment (brytebee) [inline index.html:57]: - [ ] You have properly positions the elements in this section, however, I observed that the padding added is less than the original design. Let's fix the padding to look more like the template.

Your Implementation | Figma Template
:---: | :---:
image | image

File: index.html [hunk 2]
```diff
@@ -188,21 +206,36 @@ <h2 class="heading">About</h2>
...
+              class="icon-bg"
+              src="images/image_dots_with_teal.svg"
+              alt="#"
+            />
+          </li>
         </ul>
       </section>
-      <section class="form-container">
+      <section class="form-container" id="contact">
         <form
```
Comment (brytebee) [inline index.html:219]: - [ ] **Required:** Excellent implementation of the form, however, you have not included the focus color on the border. Let's add it to signify the selected/active input element.

Your Implementation | Figma Template
:---: | :---:
image | image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Elerqsousy👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- proper GitHub flow✔️
- good commit messages✔️
- good UI design✔️
- good implementation of media query✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (MrBrN197) [inline .vscode/settings.json:3]: - [ ] [OPTIONAL] Hi you can add this file to your .gitignore as it is a vscode configuration file and doesn't need to be committed to the repo

File: index.html [hunk 4]
```diff
@@ -143,18 +159,20 @@ <h2 class="heading2">Project name goes here</h2>
...
-          aliquip ea eiusmod
-        </p>
-        <button class="buttons-primary-dark" type="button">
-          Start collaboration &#160;
-          <i class="fa fa-download" aria-hidden="true"></i>
-        </button>
+      <section class="white-bg" id="about">
+        <div class="about-container">
+          <h2 class="heading">About</h2>
+          <p class="para">
```
Comment (MrBrN197) [inline index.html:165]: - [ ] [OPTIONAL] You have a typo here in the text. this is supposed to read as get my CV.

| Figma | Your Design |
| ---| --- |
|image|image|

**Review by MrBrN197 — APPROVED** [review]
## Hi,  @Elerqsousy  & @cynthiainga 👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**